app/src/main/java/com/example/myapplication/ui/home/HomeFragment.kt

```diff
@@ -8,27 +8,50 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 import com.example.myapplication.R
+import com.example.myapplication.ui.home.recycler.PostedImageAdapter
 import kotlinx.android.synthetic.main.fragment_main.*
 
-class MainFragment : Fragment(), BaseContract.View {
+class HomeFragment : Fragment(), HomeContract.View {
 
-    lateinit var networkAdapter: NetworkAdapter
+   var postedImageAdapter: PostedImageAdapter = PostedImageAdapter(
 
-    var presenter: BaseContract.Presenter = Presenter(this)
+   )
+
+    var presenter: HomeContract.Presenter =
+        HomePresenter(this)
 
     companion object {
         @JvmStatic
         fun newInstance() =
-            MainFragment()
+            HomeFragment()
 
         const val IDENT = "MainFragment"
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+    }
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+
+        return inflater.inflate(R.layout.fragment_main, container, false)
+    }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        presenter.init()
+        rvMainPosts.apply {
```

Comment:
Warto żeby to prezenter decydował, w jakiej kolejności i co będzie inicjowane w widoku. Sekcja `rvMainPosts.apply { }` powinna znaleźć się w `fun initView()`, potrzebna jest również zmiana kolejności funkcji pochodzących z HomeContract.View